### PR TITLE
docs: dashmate logging for platform services

### PR DESCRIPTION
**Pull request by thephez**
Info on log setup for Platform services:
- [x] Gateway
- [x] Drive ABCI
- [x] Tenderdash

Preview build: https://dash-docs--396.org.readthedocs.build/en/396/docs/user/network/dashmate/index.html#configure-platform-logs

### CODE REVIEW

File: docs/user/network/dashmate/index.rst [hunk 1]
```diff
@@ -494,6 +513,166 @@ To disable logging to a file outside the container, reset the log path to ``null
...
+Configure Platform logs
+-----------------------
+
+For troubleshooting flexibility, dashmate provides independent log configuration for the Platform
+Gateway, Drive ABCI, and Tenderdash. Each service can be configured with the most helpful log level
+and output format.
+
+.. _dashmate-logs-platform-gateway:
+
+Gateway logs
```
Comment (shumkov) [inline docs/user/network/dashmate/index.rst:527]: Gateway has two different log types:
- Service logs they go to stdout and we can control verbosity level `platform.gateway.log.level` (?)
- Access logs (requests) they are configured with `platform.gateway.log.accessLogs`.

When you add stdout access log then they are mixed together in stdout.

File: docs/user/network/dashmate/index.rst [hunk 2]
```diff
@@ -494,6 +513,166 @@ To disable logging to a file outside the container, reset the log path to ``null
...
+
+Change log level
+^^^^^^^^^^^^^^^^
+
+Platform services support standard log levels. In increasing order of detail they are: ``error``,
+``warn``, ``info``, ``debug``, and `trace`. By default, logging is set to the ``info`` level. The
+log level for Platform services can be changed by using ``dashmate config set`` to update the
+following values:
+
+* platform.gateway.log.level
```
Comment (shumkov) [inline docs/user/network/dashmate/index.rst:664]: Access logs have logging level too.

File: docs/user/network/dashmate/index.rst [hunk 3]
```diff
@@ -494,6 +513,166 @@ To disable logging to a file outside the container, reset the log path to ``null
...
+
+To view the current Tenderdash log settings, run:
+
+.. code-block:: shell
+
+   dashmate config get platform.drive.tenderdash.log
+
+.. _dashmate-logs-change-level:
+
+Change log level
```
Comment (shumkov) [inline docs/user/network/dashmate/index.rst:656]: I'm not sure it's a good idea to combine all of them. It feels kind out of context and levels are different in different services (pleas check config json schema)

**Review by shumkov — CHANGES_REQUESTED** [review]

File: docs/user/network/dashmate/index.rst [hunk 4]
```diff
@@ -412,21 +429,23 @@ Historical files are each limited to 1GB.
 
 Press **Ctrl + X** to close the editor and **Y** and **Enter** save the file.
```
Comment (shumkov) [inline docs/user/network/dashmate/index.rst:430]: Prove a link to log rotate documentation

Comment (thephez) [reply docs/user/network/dashmate/index.rst:430]: You mean something like: https://manpages.ubuntu.com/manpages/xenial/man8/logrotate.8.html?

File: docs/user/network/dashmate/index.rst (hunk 2 above)
Comment (thephez) [reply docs/user/network/dashmate/index.rst:664]: I don't see a level setting specifically for [access logs in the JSON](https://github.com/dashpay/platform/blob/v1.4-dev/packages/dashmate/src/config/configJsonSchema.js#L673-L787). Does `platform.gateway.log.level` set the level for both service logs and access logs?

File: docs/user/network/dashmate/index.rst (hunk 3 above)
Comment (thephez) [reply docs/user/network/dashmate/index.rst]: Split them out into each section and mentioned which specific levels are available for each.

File: docs/user/network/dashmate/index.rst (hunk 1 above)
Comment (thephez) [reply docs/user/network/dashmate/index.rst:534]: Added info below

**Comment by thephez** [comment]
Going to merge this in. We can do a follow-up PR to clean up details if necessary.

**Pull request merged**